tutorials/vue/vue-inject.md

```diff
@@ -0,0 +1,52 @@
+Vue's provide/inject API is a neat way to send props directly
+to the component you want while avoiding prop drilling.
+What you do is you declare what variables you will send, i.e. provide,
+in the setup property and then in the component that needs them you use the inject property.
+Here is an example below:
+
+<div id = "content"></div>
+<script src="https://unpkg.com/vue@next"></script>
+<script>
+const app = Vue.createApp({
+  setup: function() {
+      const state = Vue.reactive({message: 'Hi grandma!'});
```

Comment:
Since this is reactive, it would be nice to make this example actually reactive. Try using the "row row row your boat" example from this email: https://www.getrevue.co/profile/masteringjs/issues/provide-and-inject-in-vue-367289